app/controllers/HOME2.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+class HomeController extends BaseController {
+
+	// Depending if the user is signed in or not, return the home page 
+	public function index(){
+		if( Auth::check() ) {
+			$pTitle			=	"Hud";
+			$latestTasks	=	Task::where('user_id', Auth::id())->where('state','incomplete')->orderBy('created_at', 'desc')->take(5)->get();
+            $latestCompletedTasks = Task::where('user_id', Auth::id())->where('state','complete')->orderBy('updated_at', 'desc')->take(10)->get();
+			$latestProjects	=	Project::where('user_id', Auth::id())->orderBy('created_at', 'desc')->take(5)->get();
+			$user 		    =   User::find(Auth::id());
+			$inProjects     =   $user->inProjects()->orderBy('created_at', 'desc')->take(5)->get();
+
+
+			return View::make('hud', compact('pTitle', 'latestProjects', 'latestTasks', 'latestCompletedTasks', 'inProjects'));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "pTitle" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.137:9000/coding_rules#rule_key=php%3AS1192)